archivist_samples/door_entry/README.md

```diff
@@ -0,0 +1,24 @@
+# Door entry sample
+
+Access control is not just an issue for files and computer systems: when it comes to connected dor locks it's a real world issue. Various authorities (landlords, building services, police, delivery companies) have legitimate reasons to enter the communal areas of shared buildings but the residents and owners of that building also need to be sure of their safety and privacy. How do you makes sure that privileged access to buildings is enabled whilst also holding the authorities to account and preventing abuses?
+
+RKVST Data Assurance Hub offers a solution to this problem: by ensuring that all stakeholders have transparent access to records of use of privilege, abuses are discouraged and quickly discovered. By combining virtual wold evidence (cryptographic identities, timestamps) with real-world evidence (photographs from the built-in camera from the door access system) a very strong record of when who accessed a building is maintained and made available for audit and dispute resolution.
```

Comment:
"have a transparent view of privileged access events"... "virtual world"

Reply:
Nice. done